package/setup.py

```diff
@@ -268,6 +268,13 @@ def using_clang():
     return is_clang
 
 
+def using_msvc():
```

Comment:
This is not used, but may be a useful future addition?

Reply:
unless it's necessary now or you know you'll need it in the next PR - please remove it

it's always easier to add code than work out why it exists & remove it

Reply:
Roger